sweep/tx_input_set.go

```diff
@@ -205,6 +231,9 @@ func (t *txInputSet) addToState(inp input.Input,
 	if reqOut != nil {
 		newSet.requiredOutput += btcutil.Amount(reqOut.Value)
 	}
+
+	// NOTE: `changeOutput` could be negative here if this input is using
+	// constraintsForce.
```

Comment:
Is this currently handled?

Reply:
yes - later down the pipeline we'd call `enoughInput` and add more wallet utxos if needed.